tests/cmd/cmd_test.go

```diff
@@ -114,3 +116,25 @@ func Test_builderror(t *testing.T) {
 		t.Errorf("Test Failed")
 	}
 }
+
+func Test_examples(t *testing.T) {
+	searchDir := os.ExpandEnv(ProjectPath)
+	fileList := []string{}
+	err := filepath.Walk(searchDir, func(path string, f os.FileInfo, err error) error {
```

Comment:
This should list only YAML files from `docs/examples` and from `examples`.

Sorry about that this is my fault, this wasn't clear from my previous comment :-(